fix crafting terminal crash

asdflj opened the pull request:
fix in crafting terminal use hot key to transfer current crafting terminal item to craft slot crash

Alexdoru commented on src/main/java/net/p455w0rd/wirelesscraftingterminal/common/container/ContainerWirelessCraftingTerminal.java:
```diff
@@ -1317,6 +1317,8 @@ public ItemStack slotClick(int slot, int button, int flag, EntityPlayer player)
                     && getSlot(slot) != null
                     && getSlot(slot).getStack() == RandomUtils.getWirelessTerm(player.inventory)) {
                 return null;
+            }else if(player.inventory.currentItem == button && flag == 2){
```
you need to format your code like this to pass the spotless check 
![image](https://user-images.githubusercontent.com/57050655/190576926-a3ed2c4e-6976-47ed-a310-c908b00603c4.png)

or in your IDE you can run the spotlessJavaApply task
![image](https://user-images.githubusercontent.com/57050655/190576983-cb41cb89-deeb-4173-b110-56eafbe26dc3.png)

Alexdoru commented:
that doesn't fix the crash entirely by the way, since you can also open the terminal via a keybind, and not necessarilly by holding it in your hand

eigenraven commented:
The proper way to fix it is to make the slot with the terminal a SlotLocked from the ae2 slots classes

Dream-Master commented:
@asdflj i run spotless and update Buildscript over your pr to test it. Feel free to merge this in your pr or run it on your end.
https://github.com/GTNewHorizons/WirelessCraftingTerminal/commit/27759a53436bab88318e32008ac11e2acb5d86f8

asdflj commented:
> @asdflj i run spotless and update Buildscript over your pr to test it. Feel free to merge this in your pr or run it on your end. [27759a5](https://github.com/GTNewHorizons/WirelessCraftingTerminal/commit/27759a53436bab88318e32008ac11e2acb5d86f8)

this not good solution pls do not merge

asdflj commented:
> The proper way to fix it is to make the slot with the terminal a SlotLocked from the ae2 slots classes

ur right

Dream-Master commented:
> > @asdflj i run spotless and update Buildscript over your pr to test it. Feel free to merge this in your pr or run it on your end. [27759a5](https://github.com/GTNewHorizons/WirelessCraftingTerminal/commit/27759a53436bab88318e32008ac11e2acb5d86f8)
> 
> this not good solution pls do not merge

Spotless not change any code just the format. Update Buildscript does not harm or what you mean ?

Commit:
fix crafting terminal crash
fix in crafting terminal use hot key to transfer current crafting terminal item to craft slot crash

asdflj commented:
> > > @asdflj i run spotless and update Buildscript over your pr to test it. Feel free to merge this in your pr or run it on your end. [27759a5](https://github.com/GTNewHorizons/WirelessCraftingTerminal/commit/27759a53436bab88318e32008ac11e2acb5d86f8)
> > 
> > 
> > this not good solution pls do not merge
> 
> Spotless not change any code just the format. Update Buildscript does not harm or what you mean ?

I mean,I push the code is not good solution to fix crafting terminal  crash, sorry, My english is not good

asdflj commented:
should be fine for now
